samples/shields/x_nucleo_iks01a1/src/main.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <sensor.h>
+#include <stdio.h>
+#include <misc/util.h>
+
+void main(void)
+{
+	struct sensor_value temp, hum, press;
+	struct sensor_value magn_xyz[3], accel_xyz[3];
+	struct device *hts221 = device_get_binding("HTS221");
+	struct device *lis3mdl = device_get_binding("LIS3MDL");
+	struct device *lsm6ds0 = device_get_binding("LSM6DS0");
+	struct device *lps25hb = device_get_binding("LPS25HB");
```

Comment:
Get these from the device tree labels rather than hardcoding the strings

Reply:
Indeed, would better, thanks for the review and hint! Done!